anti.md

```diff
@@ -58,12 +127,53 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think this is where this idea really falls flat on its face. I think that
+most of the work of this project would be on dealing with the course
+registration system. The language would probably some sort of cross between a
+CLI and SQL, and even if it were done graphically it would probably have a
+similar core.
+
+The result is that I think most of the time on this project would be spent
+trying to successfully interact with Portal, which is definitely on the
+**systems** side of things.
```

Comment:
While that's true, I bet you could probably put a lot of thought into the language design too.  There are a number of aspects of SQL (or at least Transact SQL, which I'm more familiar with) that are very unpleasant.